doc/Options/z390_options.md

```diff
@@ -0,0 +1,129 @@
+MAXSIZE(50)| MALE | 50 | Maximum size of any output file in MB
+MAXSYM(50000) | MA | 50000 | Maximum symbols in one macro assembly
+MAXWARN(4) | MA | 4 | Maximum MNOTE warning level without generating error.
+MAXWIDTH(800) | E | 800 | Maximum pixel width for GUI dialog windows
+MCALL | A | NO | List each macro call and exit on the PRN assembly listing in addition to first level calls if LISTCALL option is also on.
+MEM(1) | E | 1 | Memory allocated for 390 program execution in MB.  Note for RMODE31 loads and GETMAIN's memory above the 16 MB line must be allocated.  For example MEM(32) would provide 16 MB below the line and 16 MB above.
+MINHEIGHT(150) | E | 150 | Minimum pixel height for GUI dialog windows
+MINWIDTH(150) | E | 150 | Minimum pixel width for GUI dialog windows
+MNOTE(0) | MA | 0 | Control MNOTE error/warning messages during macro expansion and assembly: <br/> 0 - default generates MNOTE error/warning during mz390 and az390 <br/> 1 - suppress mz390 error/warning and only pass MNOTE to az390 <br/> 2 - generates MNOTE error/warning during mz390 and suppresses passing them on to az390
+MOD | L | NO | Generate raw code file from lz390 with .MOD suffix and no header or trailer and no RLD's. 
+OBJ | A | YES | Generate relocatable object code file
+OBJHEX | A | NO | Generate ASCII readable form of OBJ file with support for 31 bit long sections.  The default is to generate binary OBJ file which is compatible with mainframe linkers but is limited to 24 bit long sections.
+PARM('text') | E | none | Define character string parm to passed to executable program via address in R1 at startup pointing to a fullword pointing to a halfword length followed by EBCDIC characters. When the length is zero, no PARM is present.
+PC | M | YES | Generate macro pseudo code in cache memory for speeding up macro code that is executed more than once during macro expansion.
+PCOPT | M | YES | Optimize macro pseudo code by looking for push, add/sub, store sequences and replacing them with inc/dec pseudo code opcodes.  These pseudo codes are maintained in a cache in memory during macro execution only and have no affect on generated object code.
+PRINTALL | A | NO | Suppress PRINT OFF and PRINT NOGEN commands to force all source lines on PRN listing.
+PDSMEM8 | M | NO | If PDSMEM8 option is specified, error messages will be issued for any macro or copybook file names with length greater than 8.  This options is available to provide compatibility with mainframe PDS libraries that are limited to 8 character names. Note zCOBOL and zCICS use names longer than 8 characters such as zcobol\IDENTIFICATION.MAC and cics\CICS_INQUIRE.MAC.
+PROFILE(file) | M | NO | Insert COPY file in front of MLC source file
+PROLOG | M | YES | Generate CICS prolog macro calls DFHEIGBL, DFHEISTG, and  DFHEIENT if CICS option and PROLOG options are on.
+PROTECT | E | YES | Prevent modification of PSA in low memory 8k block
```

Comment:
Line 81.
1. "8k" -> "8K".
2. Missing period at end.

Reply:
According to Wikipedia https://en.wikipedia.org/wiki/Kilo- Kilo uses lower case k, not upper case. I assume thats what is meant by this statement.

Reply:
Item 2. Fixed 40c73aa

Reply:
Agreed, "8k" is shorthand for 8*1000. When referring to number of bytes, 8000 bytes = 8kB. However, the "8k" in the PROTECT table entry is referring to 8*1024 bytes, which is represented by 8KB, In both cases, kB or KB, the trailing B is almost always omitted.  See https://en.wikipedia.org/wiki/Kilobyte. In the late 1990's, new notation was introduced to avoid ambiguity. Using the new notation, 8K or 8KB is written as 8KiB = eight kibibytes.

The "8k" in the PROTECT table entry should be "8K".

Reply:
Thanks John. I learned the difference between `k` and `K` today..

I think "8 KB" is the right answer. 

From the Wiki page:

> The binary meaning of the kilobyte for 1024 bytes **typically** uses the symbol KB, with an uppercase letter K. The B is often omitted in **informal use**.

I like to think we are leaning towards the formal rather than informal notation for the documentation as a general principal. Is there a precedent in the current documentation? What does IBM use in their doco?

This is a small thing, but whatever we decide here, I will propagate through the rest of the doco so its a bigger decision that just this edit.

Reply:
Yes "8 KB" is the right answer, but in  context. where "bytes" is implied, the B is dropped and one writes "nK", not "n K". Attached is a screenshot from the IBM manual "z/Architecture Principles of Operation" where prefixing (PSA stands for Prefixed Save Area) is discussed. The IBM manual has to be precise. The PROTECT table entry is more informal and is constrained by space, so is written tersely.

It is my opinion that "8K" is best here. You might see what Don or Abe thinks.

![ibm-poo-prefixing](https://user-images.githubusercontent.com/58781735/116782711-61cf1900-aa50-11eb-80df-3bb177be78a5.png)

Reply:
Looks like they use `8 K-byte` and `8K bytes`. In both notations, byte or bytes is provided and not assumed. Maybe this is a Dan Greiner question given his editorial experience.

Reply:
I concede..  Fixed in 1957dd3. I have also added a comment in the doc contributing guide to make this the standard going forward.